src/assets/scripts/client/game/GameController.js

```diff
@@ -441,6 +441,27 @@ class GameController {
 
         return parseFloat(currentPtlVal);
     }
+    
+    /**
+     * Check whether or not the trailing distance separator should be drawn.
+     *
+     * Used by the `CanvasController` to determine whether or not to proceed with 
+     * `canvas_draw_separation_indicator`.
+     *
+     * @for GameController
+     * @method shouldUseTrailingSeparator
+     * @param aircraft
+     * @return {boolean}
+     */
+    shouldUseTrailingSeparator(aircraft) {
+        if (this.getGameOption(GAME_OPTION_NAMES.DRAW_ILS_DISTANCE_SEPARATOR) === 'yes') {
```

Comment:
Without giving you much context, this is a very good first attempt. This is exactly what I was looking for, however, the logic can be simpler. 

Instead of using two ifs, we can simply return the following:
`return this.getGameOption(GAME_OPTION_NAMES.DRAW_ILS_DISTANCE_SEPARATOR) === 'yes'`

This will give us true when yes and false when not yes. Exactly what we need!